atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/CoordinationStore.java

```diff
@@ -20,11 +20,23 @@
 import java.util.function.Function;
 
 import com.palantir.atlasdb.keyvalue.impl.CheckAndSetResult;
+import com.palantir.processors.AutoDelegate;
 
 /**
  * A {@link CoordinationStore} stores data that a {@link CoordinationService} may use.
  */
+@AutoDelegate
 public interface CoordinationStore<T> {
+    /**
+     * Coordination stores may require asynchronous initialization, if dependencies aren't initially available.
+     *
+     * @return true iff the coordination store is ready to service requests
+     */
+
```

Comment:
nit: newline

Reply:
Yep, fixed - interesting that this rendered separately from Sandor's comment